module9/ImageSearch/ImageSearchTests/ImageSearchTests.swift

```diff
@@ -0,0 +1,128 @@
+//
```

Comment:
Project deserves much more thorough tests (esp w/ view model in place and downloader), but trying to get the ball rolling w/ submission.